src/api.raml

```diff
@@ -506,29 +506,65 @@ traits:
-                                application/vnd.elife.article-poa+json;version=1:
-                                    schema: !include ../dist/model/article-poa.v1.json
+                                application/vnd.elife.article-poa+json;version=2:
+                                    (experimental):
+                                    schema: !include ../dist/model/article-poa.v2.json
                                     examples:
                                         minimum:
                                             displayName: Minimum
-                                            value: !include samples/article-poa/v1/minimum.json
+                                            value: !include samples/article-poa/v2/minimum.json
                                         complete:
                                             displayName: Complete
-                                            value: !include samples/article-poa/v1/complete.json
+                                            value: !include samples/article-poa/v2/complete.json
                                         unpublishedMinimum:
                                             displayName: PreviewMinimum
-                                            value: !include samples/article-poa/v1/unpublished-minimum.json
+                                            value: !include samples/article-poa/v2/unpublished-minimum.json
                                         unpublishedComplete:
                                             displayName: PreviewComplete
-                                            value: !include samples/article-poa/v1/unpublished-complete.json
+                                            value: !include samples/article-poa/v2/unpublished-complete.json
+
+                                application/vnd.elife.article-vor+json;version=2:
+                                    (experimental):
+                                    schema: !include ../dist/model/article-vor.v2.json
+                                    examples:
+                                        minimum:
+                                            displayName: Minimum
+                                            value: !include samples/article-vor/v2/minimum.json
+                                        complete:
+                                            displayName: Complete
+                                            value: !include samples/article-vor/v2/complete.json
+                                        unpublishedMinimum:
+                                            displayName: PreviewMinimum
+                                            value: !include samples/article-vor/v2/unpublished-minimum.json
+                                        unpublishedComplete:
+                                            displayName: PreviewComplete
+                                            value: !include samples/article-vor/v2/unpublished-complete.json
+                post:
```

Comment:
also the `post` doesn't define the body of the request, which should probably include the `queryParameters` instead (needs similar elaboration as the `put` on use cases in which these parameters are used)

Reply:
`POST` on a specific version is odd. What does it mean?